### DISCUSSION TITLE
Can't keep spawned process running in background in Windows

### DISCUSSION THREAD

**Discussion by davidmurdoch** [Microsoft/node-pty#740] [Q&A]
(aside: I think the `killDeepTree` example is broken on Windows)

I'd like to spawn a process in the background, listen in on it for a while, and then have the parent process exit. Just like setting node's `SpawnOptions` `detached` property to `true`.

Is this possible with node-pty on Windows?

**Comment by Tyriar** [comment]
I think this is essentially a feature request as we never tried to add support for this. This is out of scope for the project for now but if it's easy I'm not opposed to accepting a contribution.

**Reply by davidmurdoch** [reply]
Sure, though I'd classify as a request for parity, since it can be done on linux and macOS.
